test/auth.test.js

```diff
@@ -58,15 +67,57 @@ describe('Auth', () => {
     return client.login('email', 'password')
       .then(userId => expect(userId).toEqual('fake-user-id'));
   });
+});
 
-  it('should return a promise for login with only existing auth data userId', async() => {
-    window.fetch.resolves(mockApiResponse());
-    expect.assertions(1);
+describe('Auth login semantics', () => {
+  const email = 'test_user@domain.com';
+  const password = 'password';
+  let test = new StitchMongoFixture();
+  let th;
+  let client;
 
-    let client = new StitchClient();
-    await client.auth.storage.set(common.USER_AUTH_KEY, mockAuthData());
+  beforeAll(async() => await test.setup());
```

Comment:
[q] If `test.setup` is a promise, can't you just do `beforeAll(test.setup())`?

Reply:
Apparently, that isn't allowed.